docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,52 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(fs-legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a [native Azure
+implementation](/object-storage/file-system-azure) that does not rely on the
+legacy Hadoop implementation. Trino strongly advises upgrading existing
+deployments to the new native implementation. Legacy support will soon be
+deprecated and removed.
+
+To migrate a catalog to use the native filesystem implementation for Azure, make
+the following edits to your catalog configuration:
+
+1. Add the `fs.native-azure.enabled=true` catalog configuration property.
+2. Configure the `azure.auth-type` catalog configuration property.
```

Comment:
link to the properties for the new support either here or after the list with a explicit sentence that says something like .. for more info on the new stuff look at x